Litchi.java

```diff
@@ -1,9 +1,19 @@
 public class Litchi {
 
+private double weight; 
+
     public String toString() {
 
         return "This is a Litchi created by ZichenWang";
 
     }
+// Getter
+public double getWeight() {

```

Comment:
indentation issues, also this change should have been done in the same pull request as the whole team